.gitignore

```diff
@@ -2,4 +2,6 @@
 /target/
 **/*.rs.bk
 Cargo.lock
-*~
\ No newline at end of file
+*~
+
+.idea/
```

Comment:
The addition of `Clone` is fine but what are these changes to the .gitignore?

Reply:
It's a folder with internal settings for IDE (https://www.jetbrains.com/idea/). It mustn't be committed.